cmd/promtool/main.go

```diff
@@ -99,6 +89,30 @@ func checkConfig(t cli.Term, filename string) ([]string, error) {
 		ruleFiles = append(ruleFiles, rfs...)
 	}
 
+	found := func(fn string) bool {
+		// Nothing set, nothing to error on.
+		if fn == "" {
+			return true
+		}
+		_, err := os.Stat(fn)
+		return err == nil
+	}
+
+	for _, scfg := range cfg.ScrapeConfigs {
+		if !found(scfg.BearerTokenFile) {
+			return nil, fmt.Errorf("bearer token file %q does not exist", scfg.BearerTokenFile)
```

Comment:
The error message might be a bit misleading. Stat could fail for a lot of different reasons. Maybe use a more generic wording.